src/vivarium/framework/population/population_view.py

```diff
@@ -163,14 +163,11 @@ def get(self, index: pd.Index[int], query: str = "") -> pd.DataFrame:
         --------
         :meth:`subview <PopulationView.subview>`
         """
-        pop = self._manager.get_population(self.columns, "tracked" in self.columns)
-        pop = pop.loc[pop.index.intersection(index)]
-
-        if not index.empty:
-            if self.query:
-                pop = pop.query(self.query)
-            if query:
-                pop = pop.query(query)
+
+        combined_query = " and ".join(filter(None, [self.query, query]))
```

Comment:
What is `filter` doing here?

Reply:
It's filtering out an empty "" if one of the two queries is just the default so we don't end up w/ `tracked == True and ""`